project/src/main/webapp/events-feed.js

```diff
@@ -1,15 +1,76 @@
 $(async function() {
-  const allEvents = await getAllEvents();
-  populateAllEvents(allEvents);
+  // If want all events on discovery page 
+  if (!(window.location.href).includes('filtered=true')) {
```

Comment:
You may want to check if a parameter exists here rather than using includes. I think you can use URLSearchParams.has()

Reply:
Will keep as is just in case we want to add other params for other cases and can 'else if'